### PR TITLE
Windows and Docs

### PR DESCRIPTION
**Pull request by jwaldrop107**
Trying to test with windows. More work is still needed on `CMaize`, so shelving the changes. Pivoting to make some updates to docs and maintain some changes.

### CODE REVIEW

File: include/sigma/detail_/operation_common.hpp [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 namespace sigma::detail_ {
 
+// Value of Pi
+constexpr double pi = 3.14159265358979323846;
```
Comment (ryanmrichard) [inline include/sigma/detail_/operation_common.hpp:9]: https://stackoverflow.com/a/1727896

Comment (jwaldrop107) [reply include/sigma/detail_/operation_common.hpp:9]: I tried a couple of versions of this, but it wasn't working out.

Comment (ryanmrichard) [reply include/sigma/detail_/operation_common.hpp:9]: But hard-coding it is really going to burn you when the C++ library updates the value... :wink:

Comment (jwaldrop107) [reply include/sigma/detail_/operation_common.hpp:9]: `M_PI` isn't part of the standard, so the value is already liable to burn you depending on library. C++20 introduces the constant pi to the standard, but `sigma` currently assumes C++17. Besides, I don't foresee changes of the value of pi within the first 20 decimal places (besides precision limitations of `double`, which is lower than 20 places).

Comment (ryanmrichard) [reply include/sigma/detail_/operation_common.hpp:9]: Sorry, I meant that comment that to be sarcastic/joking. I hope Pi doesn't change...

**Pull request merged**